src/App.jsx

```diff
@@ -18,14 +19,38 @@ class App extends React.Component {
     this.setState({ productId });
   }
 
+  componentDidMount() {
+
+    const config = {
+      method: "get",
+      url: "https://app-hrsei-api.herokuapp.com/api/fec2/hr-nyc/products/39333",
+      headers: {
+        Authorization: API_KEY,
+      },
+    };
+
+    axios(config)
+      .then(function (response) {
+        console.log(JSON.stringify(response.data));
+        this.setState({ productId: response.data.id });
+      })
+
+      .catch(function (error) {
+        console.log(error);
+      });
+  }
+
   render() {
     const { productId } = this.state;
     return (
       <div>
         <Overview productId={productId} />
         <Reviews productId={productId} />
         <Questions productId={productId} />
-        <Related productId={productId} updatedProductId={this.updatedProductId} />
+        <Related
+          productId={productId}
+          updatedProductId={this.updatedProductId}
```

Comment:
I don't see where updatedProductId is defined... am I just missing something obvious?

Reply:
it happens above the axios config file, I did not make any changes to it  so it should be fine.

Reply:
Ah, I see. Github was hiding it from me

Reply:
Looking at this more I think the name updatedProductId is weak (my naming originally, sorry). It could be something like: setAppProductId, updateAppProductId, setProductDetailId or similar... Thoughts?

Reply:
Something like setProductId or updateProductId?

Reply:
@anerolabs I renamed it to updateAppProducId as that felt the most explicit and commited, should be okay now !